nomad/worker.go

```diff
@@ -349,6 +349,8 @@ func (w *Worker) UpdateEval(eval *structs.Evaluation) error {
 
 	// Store the snapshot index in the eval
 	eval.SnapshotIndex = w.snapshotIndex
+	eval.CreateTime = time.Now().UTC().UnixNano()
```

Comment:
As the method name suggests, this is updating an existing eval so you should not be setting create time here